### PR TITLE
Branch add rank command

### PR DESCRIPTION
**Pull request by victorpengmx**
Part of #81.

Changes:

- Add rankPerson command and support for rankPerson command.
- Add tests for rankPerson command.

### CODE REVIEW

File: src/main/java/connectify/logic/commands/RankPersonCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public static final String MESSAGE_SUCCESS = "Ranked all persons";
+
+    public static final String EMPTY_LIST_MESSAGE = "There are no persons in Connectify";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        Comparator<Person> comparator = Comparator.comparing(Person::rank);
+        model.updateSortedPersonList(comparator);
```
Comment (geraldngjx) [inline src/main/java/connectify/logic/commands/RankPersonCommand.java:27]: Thanks for the changes! Looks good to me! However, can I check if running the usual "list" command will display the original list order of the addressbook? Seems like the "rank" command will permanently update the order.

**Review by geraldngjx — CHANGES_REQUESTED** [review]
Looks good to me! Just need clarification on the comment that I left in the review! Thanks!

File: src/main/java/connectify/logic/commands/RankPersonCommand.java (hunk 1 above)
Comment (victorpengmx) [reply src/main/java/connectify/logic/commands/RankPersonCommand.java:27]: Great question! The 'rank' command does permanently update the order in the person list, however, the delete person and edit person command will still work, by using the new index to refer to the person to be deleted.

**Review by geraldngjx — APPROVED** [review]
LGTM!

**Pull request merged**